app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+# Landing page
```

Comment:
By signaling this controller as the `root` _(see comment below)_, we can probably get rid of this comment. The `root` nature is already signaling this via the `routes` file.

Reply:
Regarding code comments in general, I tend to lean towards not having them. 

To me, a code block which requires a comment in order to be understood is a [code smell](https://en.wikipedia.org/wiki/Code_smell). This smell probably indicates that the code is not written in a readable and easy to reason manner.